README.md

```diff
@@ -77,6 +77,43 @@ The sky's the limit when it comes to creativity! Whether you're aiming for a
 traditional system information display or something entirely unique, **pbfetch**
 empowers you to bring your vision to life.
 
+### Configuration Info
+
+Your configuration file must be placed in `$XDG_CONFIG_HOME/.config/pbfetch`, and the file itself must be called `config.txt`.
```

Comment:
This wording could be improved. It’s a bit awkward as it currently stands, so it might benefit from some rephrasing